code/modules/projectiles/guns/projectile/automatic/lmg.dm

```diff
@@ -121,7 +121,7 @@
 	icon_base = "saw"
 	icon_state = "saw_closed"
 	item_state = "saw_closed"
-	mag_well = MAG_WELL_LINKED_BOX|MAG_WELL_STANMAG
+	mag_well = MAG_WELL_LINKED_BOX|MAG_WELL_STANMAG|MAG_WELL_BOX //made it accept large boxes because it's not as good as the other machineguns
```

Comment:
```suggestion
	mag_well = MAG_WELL_LINKED_BOX|MAG_WELL_STANMAG
```
If you can slap a box in then you would never use linked ammo, please find a different way to improve this weapon

Reply:
You want more damage?

Reply:
I want to make it contrast to other machineguns by having more ammo, since it does less damage.

Reply:
Remember that guns only deal less damage TO OTHER PLAYERS, all of them are still x1 damage against mobs as gun pve damage was completely untouched, and only the projectile's damage (and gun penetration penalty) matters
However I think making them take linked mags is still a good change for an otherwise terrible gun (it's a nerfed pulemyot, which is considered our baseline since the skeleton LMG is unobtainable)

Reply:
it takes every type of 257 ammo mag and linked box, its main gimmic unlike other lmgs is that it takes mags/drums/shorts/longs/linked so you have compatable ammo with the STS or other 257 guns